web/docs/data-model/prisma-file.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Prisma File
```

Comment:
```suggestion
title: prisma.schema File
```
Maybe like this? Or you think Prisma File is understandeable?

Reply:
We could call it `Prisma Schema file` which feels more like a title. We also call the Wasp file, Wasp file vs. `main.wasp` in our docs.